### ISSUE TITLE
Command to purge your info

### ISSUE THREAD

**Issue by TranRed** [hemirt/WNMAbot#29]
20:41:42 YONN: is there a way to reset or remove your info hemirt ? pajaL
20:41:51 hemirt: no FeelsDankMan you gotta remove it one by one
20:41:55 YONN: FeelsDankMan

**Comment by hemirt** [comment]
low priority,
after #27

**Comment by hemirt** [comment]
exists
